### PR TITLE
[New Rule] Potential Reverse Shell Activity via Terminal

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/820
## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/cross-platform/execution_revershell_via_shell_cmd.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/01/07"
+maturity = "production"
+updated_date = "2020/01/07"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies the execution of a Shell process with suspicious args that may indicate a reverse shell activity."
```
Comment (bm11100) [inline rules/cross-platform/execution_revershell_via_shell_cmd.toml:8]:
```suggestion
description = "Identifies the execution of a shell process with suspicious arguments which may be indicative of reverse shell activity."
```

Comment (bm11100) [reply rules/cross-platform/execution_revershell_via_shell_cmd.toml:8]: re-lint

**Review by bm11100 — APPROVED** [review]

File: rules/cross-platform/execution_revershell_via_shell_cmd.toml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Potential Reverse Shell Activity via Terminal"
+references = [
+    """
+    https://github.com/swisskyrepo/PayloadsAllTheThings/blob/master/Methodology and Resources/Reverse Shell
+    Cheatsheet.md
+    """,
```
Comment (brokensound77) [inline rules/cross-platform/execution_revershell_via_shell_cmd.toml:18]: was this linted this way over 2 lines? IINM it should keep links in tact, even if exceeded max line width. Also, can you url encode the link with `%20`

File: rules/cross-platform/execution_revershell_via_shell_cmd.toml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+risk_score = 73
+rule_id = "a1a0375f-22c2-48c0-81a4-7c2d11cc6856"
+severity = "high"
+tags = ["Elastic", "Host", "Linux", "macOS", "Threat Detection", "Execution"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ process.name in ("sh", "bash", "zsh", "dash", "zmodload") and
+ process.args:("*/dev/tcp/*", "*/dev/udp/*", "zsh/net/tcp", "zsh/net/udp")
```
Comment (brokensound77) [inline rules/cross-platform/execution_revershell_via_shell_cmd.toml:31]: nit:

```suggestion
process where event.type in ("start", "process_started") and
  process.name in ("sh", "bash", "zsh", "dash", "zmodload") and
  process.args:("*/dev/tcp/*", "*/dev/udp/*", "zsh/net/tcp", "zsh/net/udp")
```

**Review by brokensound77 — APPROVED** [review]
A couple nits, but otherwise LGTM 👍

File: rules/cross-platform/execution_revershell_via_shell_cmd.toml (hunk 2 above)
Comment (Samirbous) [reply rules/cross-platform/execution_revershell_via_shell_cmd.toml]: encoded.

**Pull request merged**